packages/patternfly-4/react-styled-system/src/components/StyledSystem/StyledTheme.d.ts

```diff
@@ -161,7 +162,7 @@ export interface StyledConstants {
 }
 
 export interface PatternFlyThemeProviderProps extends HTMLProps<HTMLDivElement> {
-  theme?: Object;
+  theme?: object;
```

Comment:
Shouldn't this be `Object` not lowecase

Reply:
@seanforyou23 pointed out [above](https://github.com/patternfly/patternfly-react/pull/3719#discussion_r379619119) that typically [Object is used incorrectly and object is preferred](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#general-types)